packages/nx/src/utils/package-manager.spec.ts

```diff
@@ -10,6 +10,9 @@ import {
 
 describe('package-manager', () => {
   describe('detectPackageManager', () => {
+    afterEach(() => {
+      jest.restoreAllMocks();
```

Comment:
Without this it appears these tests are not logically because it increase the call and depends on the state. Doing this allows you to predict correctly whats changing.